couchbase-on-ubuntu/azuredeploy.json

```diff
@@ -69,27 +69,27 @@
     },
     "cbPackageDownloadBase": {
       "type": "string",
-      "defaultValue": "http://packages.couchbase.com/releases/3.0.3/",
+      "defaultValue": "http://packages.couchbase.com/releases/4.1.1/",
       "metadata": {
         "description": "Couchbase package download location"
       }
     },
     "cbPackage": {
       "type": "string",
-      "defaultValue": "couchbase-server-enterprise_3.0.3-ubuntu12.04_amd64.deb",
+      "defaultValue": "couchbase-server-enterprise_4.1.1-ubuntu12.04_amd64.deb",
```

Comment:
The template above (checkpoint) shows how it offers two different images based on user input. Here's another one https://github.com/Azure/azure-quickstart-templates/blob/master/create-hpc-cluster/azuredeploy.json#L179. I'll merge this PR and you can make this modification in the next one.